<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_Background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1500</width>
      <height>800</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>2</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tranning</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dataset_IMG">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>381</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#dataset_IMG{
	border: 1px solid black;
	border-radius: 15px
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Tranning">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>330</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Tranning</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_Employee">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>220</y>
      <width>651</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#tbl_Employee{background: transparent;}</string>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Full Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Department</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dataset</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="btn_Back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>2</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../public/img/back.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Dataset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>700</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HUTECH</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Close">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>2</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../public/img/close.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1500</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lbl_Header{background-color:#248f8f }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Minimize">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>2</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../public/img/minimize.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>175</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Employee List</string>
    </property>
   </widget>
   <zorder>lbl_Background</zorder>
   <zorder>dataset_IMG</zorder>
   <zorder>btn_Tranning</zorder>
   <zorder>tbl_Employee</zorder>
   <zorder>label_2</zorder>
   <zorder>label_Text</zorder>
   <zorder>lbl_Header</zorder>
   <zorder>lbl_Close</zorder>
   <zorder>btn_Back</zorder>
   <zorder>label</zorder>
   <zorder>lbl_Minimize</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
